tools/ci_build/builds/wheel_verify.sh

```diff
@@ -19,9 +19,8 @@ set -e
 if [[ $(uname) == "Darwin" ]]; then
     CMD="delocate-wheel -w wheelhouse"
 else
-    LD_PATH="$(cat .bazelrc | grep TF_SHARED_LIBRARY_DIR | sed 's/"//g' | awk -F'=' '{print $2}')"
-    export LD_LIBRARY_PATH=$LD_LIBRARY_PATH:$LD_PATH
-    CMD="auditwheel repair --plat manylinux2010_x86_64"
+    pip3 install auditwheel==2.0.0
```

Comment:
Can we use pip here?

Reply:
So auditwheel==1.5.0 (too old for manylinux2010) is by default installed on py36. I've added a new commit to specifically update the py36 version.

Also the patch will only get called once now instead of each time auditwheel is ran.